src/components/blog/blog-page.js

```diff
@@ -15,26 +15,41 @@ class Blog extends Component {
 
   render() {
     if (this.props.apiBlogData.loaded) {
-      let MEDIUM_PUBLICATION = this.props.apiInfoData.footerData.mediumSlug
-      return (
-        <Container styleName="common.margin">
-          <Card.Group itemsPerRow={3} stackable doubling>
-            {this.props.apiBlogData.data.map((info, id) => (
-              <BlogDetail info={info} key={id} />
-            ))}
-          </Card.Group>
-          <Segment basic padded textAlign="center">
-            <Icon
-              name="medium"
-              size="large"
-              link={true}
-              onClick={() =>
-                window.open(`${MEDIUM_URL}${MEDIUM_PUBLICATION}`, '_blank')
-              }
-            />
-          </Segment>
-        </Container>
-      )
+      if (this.props.apiBlogData.data && this.props.apiBlogData.data.length) {
```

Comment:
Just check `this.props.apiBlogData.data.length > 0`.

Reply:
but if in case the apiBlogData.data is undefined then the whole code will break. So that's why I added the extra condition for that